CONTRIBUTING.md

```diff
@@ -78,6 +78,18 @@ move forward.
   The AsyncAPI spec, despite describing technical behavior, is intended to be
   read by people. Use natural tone and include motivation and examples.
 
+## Breaking Change vs Non Breaking Change
+
+The specification is not much different than software. Some changes provided in the spec can cause breaking changes for tools that support it. For example, if one of the properties that was `required` becomes `optional`, it is considered a breaking change because some tools might depend on that property and fail if all suddenly it is not provided. 
+
+_Non-breaking changes:_
+- Adding a property
+
+_Breaking changes:_
+- Making a property required
+- Making a property optional
+- Removing a property
+- Changing the type of a property in any way
```

Comment:
Is it intended to be an exhaustive list or just an example?

Reply:
from the suggestion made I think it is a list, I can't be sure of how exhaustive it is nor if it is an example.
Maybe @derberg could explain that.

Reply:
it is intended as a result of https://github.com/asyncapi/spec/issues/688

it should be explicit